assets/training/model_management/environments/foundation-model-inference/context/requirements.txt

```diff
@@ -1,17 +1,21 @@
-pandas==2.0.3
-numpy==1.25.0
-text-generation==0.6.0
-azureml-inference-server-http==0.8.4
-azureml-core==1.52.0
-azureml-contrib-services==1.52.0
-psutil==5.9.5
-mlflow==2.3.1
-azure-ai-contentsafety==1.0.0b1
-aiolimiter==1.1.0
-azureml-ai-monitoring==0.1.0b3
-azure-mgmt-cognitiveservices==13.4.0
-azure-identity==1.13.0
-cryptography==41.0.3
-certifi==2023.07.22
-requests==2.31.0
-aiohttp==3.8.5
\ No newline at end of file
+deepspeed~=0.10.1
+git+https://github.com/microsoft/DeepSpeed-MII.git@mrwyattii/fix-missing-hostfile
+aiolimiter~=1.1.0
+torch~=2.0.1
+protobuf~=4.23.4
+psutil~=5.9.5
+transformers~=4.31.0
+sentencepiece~=0.1.99
+xformers~=0.0.20
```

Comment:
xformers 0.0.21 is out, we should test with that version (not a requirement for this PR)